assets/app/views/browse/pod.html

```diff
@@ -80,11 +80,15 @@
                 <annotations annotations="pod.metadata.annotations"></annotations>
               </tab>
 
-              <!--
-              <tab heading="Metrics" active="selectedTab.metrics">
-              Metrics content
+              <tab ng-if="metricsAvailable && pod.status.phase === 'Running'" heading="Metrics" active="selectedTab.metrics">
```

Comment:
The data exists even after the pod is deleted. Metric data is stored with a time-to-live and its only then that the data will become unavailable.